app/models/compute_resources/foreman/model/ovirt.rb

```diff
@@ -6,6 +6,7 @@ class Ovirt < ComputeResource
     validates :url, :format => { :with => URI.regexp }
     validates :user, :password, :presence => true
     before_create :update_public_key
+    before_save :update_available_operating_systems
```

Comment:
quiestion: Maybe it would be better to build data in before_validation so validation can still verify attrs contents. I'm not sure if we run `valid?` without save anywhere, if we do it would slow down the call unnecessarily.